src/helpers/scrollBehavior.ts

```diff
@@ -0,0 +1,64 @@
+import { Node, ReactFlowProps } from 'react-flow-renderer';
+import { ScrollScheme } from 'index';
+
+export type Extent = ReactFlowProps['translateExtent'];
+
+export function scrollProps(scheme: ScrollScheme) {
+  let props;
+
+  switch (scheme) {
+    case 'default':
+      props = {
+        panOnScroll: true,
+        panOnScrollSpeed: 1,
+      };
+      break;
+    case 'sellkit':
+      props = {
+        panOnScroll: false,
+        zoomOnScroll: false,
+        preventScrolling: false,
+      };
+      break;
+  }
+
+  return props;
+}
+
+export function calcFlowExtent(nodes: Node[], scheme: ScrollScheme): Extent {
+  const w = window.innerWidth;
+  const h = window.innerHeight;
+  // TODO: Do the following with "ref" instead of DOM API.
+  const containerHeight = document
+    .getElementsByClassName('artbees-flow__LayoutFlow')[0]
+    .getBoundingClientRect().height;
```

Comment:
Was there any problems with using ref that you marked it as TODO? because this logic misbehaves if for whatever reason there are 2 separate `ArtbeesFlow` components at the same time. Probably not a real issue now but it can be a painful bug in the future.